lusidtools/cocoon/utilities.py

```diff
@@ -836,6 +836,11 @@ def handle_nested_default_and_column_mapping(
                 )
 
         elif isinstance(value, str):
+
+            if len(value) == 0:
+                raise IndexError(f"Unspecified mapping field: {value}. Please assign a value or remove this from the "
```

Comment:
I think you want this to be {key} instead of value as this error will only ever show nothing.

Reply:
Agreed